tests/test_agents.py

```diff
@@ -242,7 +242,10 @@ async def test_get_directory_index_w_manifest(agent_test_settings: Settings) ->
 
         results = await index.query(query="who is Frederick Bates?")
         top_result = next(iter(results[0].docs.values()))
-        assert top_result.dockey == md5sum(abs_paper_dir / "bates.txt")
+
+        # note: we get every possible field from the manifest constructed in maybe_get_manifest,
+        # so the dockey will now be equal to the doc_id.
```

Comment:
This comment doesn't make sense because `doc_id`/`doc_key` isn't in the manifest

Let's modify this comment to reflect that

```suggestion
        # note: we get every possible field from the manifest constructed in maybe_get_manifest,
        # and then DocDetails construction sets the dockey to the doc_id.
```